backend/Models/ParcelDetail.cs

```diff
@@ -56,6 +58,43 @@ public ParcelDetail(Pims.Api.Data.Entities.Parcel parcelDetail)
       this.PropertyStatus = parcelDetail.Status?.Name;
       this.Agency = parcelDetail.Agency?.Name;
     }
-    #endregion
-  }
+
+        public override bool Equals(object obj)
```

Comment:
Need to start adding comments to describe the purpose/function of the methods.  They're useful when using intellisense.

Reply:
For the equals method? I'm not sure that I agree. This is generated code for starters, and the equals method is one of the basic building blocks of OOP in my opinion. Perhaps you just mean in general?

Reply:
In general we should provide a comment for functions so that intelisense provides some useful information.  I know this is a basic override, but the default behaviour is actually very different than what you have provided here.

Reply:
Well, I'll at least check and see if the code generator can provide a comment as well the next time I do this.